packages/esm-patient-test-results-app/src/trendline-new/trendline.component.tsx

```diff
@@ -0,0 +1,206 @@
+    },
+    [setRange, upperRange, lowerRange],
+  );
+
+  /**
+   * reorder svg element to bring line in front of the area
+   */
+  useLayoutEffect(() => {
+    const graph = document.querySelector('g.bx--cc--area')?.parentElement;
+    if (obs && obs.length && graph) {
+      graph.insertBefore(graph.children[3], graph.childNodes[2]);
+    }
+  }, [obs]);
+
+  const data: Array<{
+    date: Date;
+    value: number;
+    group: string;
+    min?: number;
+    max?: number;
+  }> = [];
+
+  const tableData: Array<{
+    date: string;
+    time: string;
+    value: number;
+    interpretation?: OBSERVATION_INTERPRETATION;
+  }> = [];
+
+  const dataset = chartTitle;
+
+  obs.forEach((obs) => {
+    const range =
+      hiNormal && lowNormal
+        ? {
+            max: hiNormal,
+            min: lowNormal,
+          }
+        : {};
+
+    data.push({
+      date: new Date(Date.parse(obs.obsDatetime)),
+      value: obs.value,
+      group: chartTitle,
+      ...range,
+    });
+
+    tableData.push({
+      date: formatDate(parseDate(obs.obsDatetime)),
+      time: formatTime(parseDate(obs.obsDatetime)),
+      value: obs.value,
+      interpretation: obs.interpretation,
+    });
+  });
+
+  const options = useMemo<LineChartOptions>(
+    () => ({
+      bounds: {
+        lowerBoundMapsTo: 'min',
+        upperBoundMapsTo: 'max',
+      },
+      axes: {
+        bottom: {
+          title: bottomAxisTitle,
+          mapsTo: 'date',
+          scaleType: ScaleTypes.TIME,
+          ticks: {
+            rotation: TickRotations.ALWAYS,
+            // formatter: x => x.toLocaleDateString("en-US", TableDateFormatOption)
+          },
+          domain: range,
+        },
+        left: {
+          mapsTo: 'value',
+          title: leftAxisTitle,
+          scaleType: ScaleTypes.LINEAR,
+          includeZero: false,
+        },
+      },
+      height: '20.125rem',
+
+      color: {
+        scale: {
+          [chartTitle]: '#6929c4',
+        },
+      },
+      points: {
+        radius: 4,
+        enabled: true,
+      },
+      legend: {
+        enabled: false,
+      },
+      tooltip: {
+        customHTML: ([{ date, value }]) =>
+          `<div class="bx--tooltip bx--tooltip--shown" style="min-width: max-content; font-weight:600">${value} ${leftAxisTitle}<br>
+          <span style="color: #c6c6c6; font-size: 0.75rem; font-weight:400">${formatDate(date)}</span></div>`,
+      },
+    }),
+    [bottomAxisTitle, leftAxisTitle, range, chartTitle],
+  );
+
+  const tableHeaderData = useMemo(
+    () => [
+      {
+        header: t('date', 'Date'),
+        key: 'date',
+      },
+      {
+        header: t('value', 'Value') + ` (${leftAxisTitle})`,
+        key: 'value',
+      },
+      {
+        header: t('timeOfTest', 'Time of Test'),
+        key: 'time',
+      },
+    ],
+    [leftAxisTitle, t],
+  );
+
+  if (isLoading) {
+    return <p>Loading...</p>;
```

Comment:
Translate

Reply:
It was put up for the testing purposes only, I'll be adding a proper component here.